tests/queries/0_stateless/helpers/client.py

```diff
@@ -16,7 +16,7 @@ class client(object):
     def __init__(self, command=None, name="", log=None):
         self.client = uexpect.spawn(["/bin/bash", "--noediting"])
         if command is None:
-            command = os.environ.get("CLICKHOUSE_BINARY", "clickhouse") + " client"
+            command = os.environ.get("CLICKHOUSE_BINARY", "clickhouse") + " client --progress"
```

Comment:
Why? It will break integration tests output

Reply:
this and couple other tests were failing:
https://github.com/ClickHouse/ClickHouse/blob/master/tests/queries/0_stateless/01921_test_progress_bar.py
how they worked before then?

Reply:
Hm, sorry, I got confused, though this `client.py` is for integration tests, because functional tests on python are unusual. But seems like `01921_test_progress_bar.py ` runs client in interactive mode, so it should show progress without additional parameters.

Reply:
I and @KochetovNicolai decided that something have changed regarding interactive/non-interactive run since Alexey's changes. Let's wait tests and then I fix whatever was broken...